includes/MediaWiki/ExternalLinks.php

```diff
@@ -39,10 +40,12 @@ public function pageIdsContainingUrl( string $url ) : array {
 	 * @param string $url
 	 * @return string the el_index value
 	 */
-	private function getSingleIndexOrDie( string $url ) : string {
+	private function getSingleIndexOrThrowException( string $url ) : string {
 		$indexes = LinkFilter::makeIndexes( $url );
 		if ( count( $indexes ) !== 1 ) {
-			die( 'Unexpected issue with LinkFilter return' );
```

Comment:
I think we don’t need to make this message translatable, actually. This isn’t really a case of a potential user error – if the returned array has more than one element, then the MediaWiki method has behaved differently than we expect. I think `Assert::invariant()` might be the most appropriate method here.

Reply:
Changed.